### DISCUSSION TITLE
InputNumber组件

### DISCUSSION THREAD

**Discussion by zhenyulei** [jdf2e/nutui-react#25] [General]
支持小数时，初始值为 1.5，step="0.12" decimalPlaces="2"，最小只能到 0.9
同样上述 max="2" 时，最大值到了 2.10
文档不全，比如一些参数没有写 focus\blur\add 等事件

**Comment by szg2008** [comment]
文档 安装时的包名需要修改“@nutui/nutui-react”

**Comment by junjun666** [comment]
@zhenyulei 上述三个问题已修改
@szg2008 包名问题已修改
